Poker.BE/Poker.BE.Domain.Tests/Game/RoomTests.cs

```diff
@@ -30,5 +32,78 @@ public void RoomTest()
             TestContext.WriteLine("end of roomTest");
         }
 
+        [TestMethod()]
+        //[ExpectedException(typeof(NotEnoughPlayersException))]
+        public void StartNewHandTest()
+        {
+            //Arrange
+            Player player1 = new Player();
+            GamePreferences preferences = new GamePreferences();
+            GameCenter center = new GameCenter();
+            Room room = new Room(player1, preferences);
+            Exception expectedExcetpion = null;
+            //Act
+            try
+            {
+                room.StartNewHand();
+            }
+            catch (NotEnoughPlayersException ex)
```

Comment:
no way. the "Normal" way to do so is "[ExpectedException(typeof(NotEnoughPlayersException))]", but this option will limited to whole testmethod, so its not helping here